### PR TITLE
Merge changes to prod

### PR DESCRIPTION
**Pull request by NyAndoMayah**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -44,7 +45,15 @@ const App = () => (
       <Route exact path='/profile' element={<profile.show />} />
 
       <Route exact path='/students/:studentId/fees' element={<fees.list />} />
-      <Route exact path='/students/:studentId/fees/create' element={<fees.create />} />
+      <Route
+        exact
```
Comment (JrManjato) [inline src/App.js:49]: Put in one line "exact path"

File: src/operations/fees/FeeList.js [hunk 2]
```diff
@@ -37,12 +37,12 @@ const FeeList = ({ studentId }) => {
       title={`Frais de ${studentRef}`}
       resource={'fees'}
       label='Frais'
-      actions={<Actions basePath={`/students/${definedStudentId}/fees`} />}
+      actions={role === 'MANAGER' ? <Actions basePath={`/students/${definedStudentId}/fees`} /> : null}
```
Comment (JrManjato) [inline src/operations/fees/FeeList.js]: use:

role === 'MANAGER' && <Actions basePath={`/students/${definedStudentId}/fees`} />

instead

File: src/operations/fees/FeeList.js [hunk 3]
```diff
@@ -37,12 +37,12 @@ const FeeList = ({ studentId }) => {
...
       filterDefaultValues={{ studentId: definedStudentId }}
       pagination={false}
       perPage={maxPageSize}
     >
-      <Datagrid rowClick={id => `/fees/${id}/show`} rowStyle={rowStyle}>
+      <Datagrid bulkActionButtons={role === 'MANAGER'} rowClick={id => `/fees/${id}/show`} rowStyle={rowStyle}>
         <CustomDateField label='Date limite' source='due_datetime' />
```
Comment (JrManjato) [inline src/operations/fees/FeeList.js]: source before label

File: src/operations/fees/FeeList.js (hunk 2 above)
Comment (NyAndoMayah) [reply src/operations/fees/FeeList.js:40]: You forgot the ``. I just put it like that : role === 'MANAGER' && <Actions basePath={`/students/${definedStudentId}/fees`} />
Is it right?

**Review by JrManjato — APPROVED** [review]

**Review by JrManjato — APPROVED** [review]

**Pull request merged**